### DISCUSSION TITLE
Some problems with "Tome of Divinity"

### DISCUSSION THREAD

**Discussion by Granvisirdelcaos** [chromiecraft/chromiecraft#2384] [General]
Greetings.
I recently made a dwarf paladin. At level 12 I was given the paladin quest "The tome of divinity" but accidentally deleted the book to continue the quest.
How do I get it back? Without that I can't resurrect other players.

**Comment by 55Honey** [comment]
Hi Granvirsirdelcaos!
There's a known bug with the dwarf paladin quest. Please make a Ticket in discord, so we can interact and help you out ingame, before the fix is live.
